src/main/java/com/descope/sdk/mgmt/impl/TenantServiceImpl.java

```diff
@@ -161,6 +161,9 @@ public void configureSettings(String id, TenantSettings settings) throws Descope
     if (settings == null) {
       throw ServerCommonException.invalidArgument("settings");
     }
+    if (settings.getAuthType() == null && settings.getJitDisabled() != null) {
```

Comment:
why do you need this condition?

Reply:
So no-one by mistake will specify JIT without auth type, I guess